src/components/Header.js

```diff
@@ -87,12 +82,21 @@ export default class Header extends Component {
             {
               siteID &&
               <div className="header__site-logo">
-                <img src={process.env.PUBLIC_URL + "/assets/"+siteID+"/images/logo.png"} alt="site logo" onLoad={this.handleImageLoaded} onError={this.handleImageLoadError}/>
+                <img src={process.env.PUBLIC_URL + "/assets/"+siteID+"/images/logo.png"} siteID={siteID} alt="site logo" onLoad={this.handleImageLoaded} onError={this.handleImageLoadError}/>
               </div>
             }
-            <div className="entries">
-              <img src={process.env.PUBLIC_URL + "/assets/images/doh_logo.png"} alt="doh logo" className="default-logo"/>
-            </div>
+            {
+              !siteID &&
```

Comment:
I'd rather have a `siteID` for `demo` (as the default), and pull demo in with the same code as you do other sites.

Reply:
thanks.  I have made the change.  If the siteID is `demo` the associated logo will be loaded.